### DISCUSSION TITLE
Example JSON should be pretty printed / formatterd

### DISCUSSION THREAD

**Discussion by cc-stjm** [jy95/docusaurus-json-schema-plugin#393] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
When showing examples within the schema, they are currently output as a single line of JSON, which may be rather long.

**Describe the solution you'd like**
It would seem helpful to pretty-print the examples somehow. Color would be nice to have, but just adding newlines / indentation would be an easy win.

**Additional context**

I'm getting my head around the codebase - I think the change needs to go [here](https://github.com/jy95/docusaurus-json-schema-plugin/blob/main/src/theme/JSONSchemaViewer/utils/QualifierMessages/Examples.tsx)? 

Is it as simple as adding `JSON.stringify(item.value, null, 4)` to the `<CreateValue>` on line 53?

**Comment by jy95** [comment, marked as answer]
Hello @cc-stjm,

I suggest you look for `ValueComponent` [example](https://jy95.github.io/docusaurus-json-schema-plugin/docs/demo-viewer/custom/values) which gives you the control you seek. 

We got this reflexion in https://github.com/jy95/docusaurus-json-schema-plugin/pull/224 as possibilities are endless : 
- `JSON.stringify(value, null, 2)`
- `JSON.stringify(value, null, 4)`
- `JSON.stringify(value, null, "\t")`
- ....

**Reply by cc-stjm** [reply]
Thank you, that works
